gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +58,16 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+@pytest.mark.xfail
+@requires_data('hess')
+def test_DataStore_load_all(data_manager):
```

Comment:
xfail is not the same as fail:
https://pytest.org/latest/skipping.html#mark-a-test-function-as-expected-to-fail

Those are tests that we marked as xfail because there was some issue.
The tests need to be fixed and the xfail decorator removed.
This is not urgent or related to this PR.